acres/fastngram/fastngram.py

```diff
@@ -150,7 +152,7 @@ def _find_contexts(acronym: str, min_freq: int) -> 'List[topic_list.Acronym]':
     :param min_freq:
     :return:
     """
-    model = resource_factory.get_center_map(functions.partition(acronym))
+    model = resource_factory.get_center_map(functions.partition(acronym), PARTITIONS)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Too many positional arguments for function call](https://app.codacy.com/manual/michelole/acres/pullRequest?prid=5472465)